bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -50,16 +62,56 @@ spec:
           echo "$(date -Iseconds) - starting ovs-daemons"
           chown -R openvswitch:openvswitch /run/openvswitch
           chown -R openvswitch:openvswitch /etc/openvswitch
+          if [[ -d /run/openvswitch ]]; then
+            ls -al /run/openvswitch
+          fi
+          if [[ -d /var/run/openvswitch ]]; then
+            ls -al /var/run/openvswitch
+          fi
+          if [[ -f /var/run/openvswitch/ovs-vswitchd.pid ]] ; then
+            cat /var/run/openvswitch/ovs-vswitchd.pid
+          fi
+          if [[ -f /var/run/openvswitch/ovsdb-server.pid ]] ; then
+            cat /var/run/openvswitch/ovsdb-server.pid
+          fi
           function quit {
               # Don't allow ovs-vswitchd to clear datapath flows on exit
               kill -9 $(cat /var/run/openvswitch/ovs-vswitchd.pid 2>/dev/null) 2>/dev/null || true
               kill $(cat /var/run/openvswitch/ovsdb-server.pid 2>/dev/null) 2>/dev/null || true
               exit 0
           }
           trap quit SIGTERM
+          if [[ -S /var/run/openvswitch/db.sock ]]; then
+            echo "Found /var/run/openvswitch/db.sock"
+          else
+            echo "File /var/run/openvswitch/db.sock was not found"
+          fi
+          if [[ -S /run/openvswitch/db.sock ]]; then
+            echo "Found /run/openvswitch/db.sock"
+          else
+            echo "File /run/openvswitch/db.sock was not found"
+          fi
+          export SYSTEMD_IGNORE_CHROOT=yes
+          journalctl -xeu ovsdb-server --no-pager
+          #systemctl is-active --quiet ovsdb-server
+          journalctl -xeu openvswitch --no-pager
+          if (modprobe openvswitch) ; then
+            echo "modprobe openvswitch success?"
+          else
+            echo "modprobe openvswitch fail?"
+          fi
+          lsmod | grep openvswitch
+          if (systemctl is-active --quiet openvswitch) ; then
+            echo "OVS started by RHCOS"
+            systemctl is-active openvswitch
+            journalctl -xeu openvswitch --no-pager
```

Comment:
Probably just want to `exec journalctl -f -u openvswitch` and be done with the script.